### ISSUE TITLE
Is ab-p-document029.xml correct?

### ISSUE THREAD

**Issue by ndw** [xproc/3.0-test-suite#41]
I've now implemented JSON-to-XML through `fn:json-to-xml`. The result I get is simply
```xml
<map xmlns="http://www.w3.org/2005/xpath-functions"><string key="key">value</string></map>
```
Where does the `c:data` wrapper and other furniture come from? (What part of the spec am I forgetting 😄 ?)

**Comment by xml-project** [comment]
The test is obviously wrong: The result you get is the same, I would expect. 
What the test does is the "casting from non-XML mediatype to XML mediatype" (which is correct according to the words of the specs).
Will change it in about an hour or so.

**Comment by xml-project** [comment]
Fixed with pr #43
